doc/field_guide__periodic.md

```diff
@@ -0,0 +1,30 @@
+## Periodic Variables (periodic)
+Periodic Variables are objects that exhibit repeating sequences of values over a fixed length of time, known as the period.
+Many astrophysical objects exhibit periodic behavior, including eclipsing binaries, pulsators, etc.
+
+### Classification and numbers
+- Supertypes
+  - variable
+- Subtypes
+  - Eclipsing
+  - Sinusoidal
+  - Sawtooth
+  - Elliptical
+
+### ZTF light curves
+![ZTF periodic](data/periodic.png)
+
+#### Description
+Periodic variables have periods ranging from minutes to over a thousand days with millimagnitude to magnitude level variations.
+
+#### Light curve characteristics
+- periodic variable
+- period range: 0.001-1000 days
+- amplitude: varies from survey precision to more than a magnitude

```

Comment:
Wording? Reminds me of an old Russian joke: new recruits should dig a trench from here to lunchtime.

Reply:
alright see what you think, i added a definition afterwords.

Reply:
Oops, that was fine, I meant line 23 re amplitude

Reply:
I guess it's fine :)